prime-router/metadata/hl7_mapping/resources/MessageHeader/MSH.yml

```diff
@@ -30,23 +30,20 @@ elements:
     constants:
       hl7HDField: 'MSH-6'
 
-  - name: datetime-of-message
```

Comment:
Nit: I'm actually wondering if we should keep this one and delete the other two?  It looks like they all come from MSH.7 and this extension will always get set if that is valued.

Reply:
Yep that works for HL7 senders, but it would change the requirements of FHIR senders, in particular some of these changes were needed for the OML FHIR sender tests. We can make those changes via transforms and go that route though, do you think that's the way we should go?

Reply:
That's my thought?  Though might not be a huge deal, but the case I was guarding against was MSH.7 not being set in the original HL7 and we then populate it on the way out